webapp/admin/views.py

```diff
@@ -102,7 +102,15 @@ def post_manage_store_snaps(store_id):
 
 @admin.route("/admin/<store_id>/members")
 @login_required
-def get_manage_members(store_id):
+def get_manage_members(
+    store_id,
+    confirm_invite=False,
+    email_address=None,
+    admin=False,
+    review=False,
+    view=False,
+    access=False,
```

Comment:
All these arguments are not possible to modify. `store_id` make sense because is a variable in the URL `/admin/<store_id>/members`.